api/room/schema.py

```diff
@@ -116,6 +116,7 @@ class Arguments:
         capacity = graphene.Int()
         image_url = graphene.String()
         calendar_id = graphene.String()
+        firebase_token = graphene.String()
```

Comment:
Hey @RachaelNantale, Nice work, I see you included the fire_base token field in the `UpdateRoom` class enabling the field to be editable, don't we need to do the same in `CreateRoom class`. If not please throw more light?

Reply:
Thanks, @nakatuddesuzan, So firstly, the story did not include the addition of the column to  the `CreateRoom class.`
Secondly, the firebase token is generated from the devices but under certain circumstances, it has to be updated. so there is no need to add it to the `CreateRoom class`. I hope this gives you more clarity. Please refer to the Android team for more information